README.md

```diff
@@ -114,6 +114,8 @@ This information is then expounded on with additional columns and tables that ma
 
 ### [Latest Schema](/db/chainquery_schema.sql)
 
+![Latest Schema](https://spee.ch/@SK3LA:3/chainqueryschema2.svg)
```

Comment:
hmm I was avoiding this in the repo because it has to be maintained. Right now, I still have to manually export the schema sql when it changes. People don't like out of date information. Can we find a way to automate this? There is also an issue in the repo for this.

Reply:
For now, lets remove this, until we can automate it.